trillian/migrillian/README.md

```diff
@@ -0,0 +1,19 @@
+Migrillian Tool
+===============
+
+*Migrillian* is a tool that transfers data from Certificate Transparency logs to
+Trillian *PREORDERED_LOG* trees.
+
+It can be used for:
+ - One-off data migrations, e.g. from legacy CT implementation to the new
+   Trillian-based solution which is this repository.
```

Comment:
Better to link to the repo in case this file is copied to somewhere else.

Reply:
Done.